gcp_variant_transforms/libs/vcf_reserved_fields.py

```diff
@@ -22,38 +22,36 @@
 import collections
 from typing import Optional  # pylint: disable=unused-import
 
-from vcf import parser
-
 _ReservedDefinition = collections.namedtuple('ReservedDefinition',
                                              ['id', 'num', 'type', 'desc'])
 
 
 def _get_field_count(value):
   # type: (str) -> Optional[int]
-  return parser.field_counts[value]
+  return value
 
 
 INFO_FIELDS = {
     'AA': _ReservedDefinition('AA', 1, 'String', 'Ancestral allele'),
-    'AC': _ReservedDefinition('AC', _get_field_count('A'), 'Integer',
+    'AC': _ReservedDefinition('AC', 'A', 'Integer',
```

Comment:
Let's define the following 3 consts, again these are duplicated definitions but later we can unify them:
```
_FIELD_COUNT_ALTERNATE_ALLELE = 'A'
_FIELD_COUNT_ALL_ALLELE = 'R'
_FIELD_COUNT_GENOTYPE = 'G'
```

Reply:
Ack.